lighthouse-core/audits/bootup-time.js

```diff
@@ -144,46 +61,47 @@ class BootupTime extends Audit {
    */
   static audit(artifacts) {
     const trace = artifacts.traces[BootupTime.DEFAULT_PASS];
-    const executionTimings = BootupTime.getExecutionTimingsByURL(trace);
+    return artifacts.requestDevtoolsTimelineModel(trace).then(devtoolsTimelineModel => {
+      const executionTimings = BootupTime.getExecutionTimingsByURL(devtoolsTimelineModel);
+      let totalBootupTime = 0;
+      const extendedInfo = {};
+
+      const headings = [
+        {key: 'url', itemType: 'url', text: 'URL'},
+        {key: 'scripting', itemType: 'text', text: groups.scripting},
+        {key: 'scriptParseCompile', itemType: 'text', text: groups.scriptParseCompile},
+      ];
 
-    let totalBootupTime = 0;
-    const extendedInfo = {};
+      // map data in correct format to create a table
+      const results = Array.from(executionTimings).map(([url, groups]) => {
```

Comment:
a little confusing that this has the same name as the outer scoped `groups`

wdyt about naming the `groups` in lib/task-groups to something like `groupIdToName` or similar?

Reply:
what about something like `groupsEnum`?

Reply:
+1 on groupsEnum.